backends-clickhouse/src/main/scala/org/apache/gluten/backendsapi/clickhouse/CHSparkPlanExecApi.scala

```diff
@@ -849,10 +849,7 @@ class CHSparkPlanExecApi extends SparkPlanExecApi with Logging {
     }
   }
 
-  /** Clickhouse Backend only supports part of filters for parquet. */
-  override def postProcessPushDownFilter(
-      extraFilters: Seq[Expression],
-      sparkExecNode: LeafExecNode): Seq[Expression] = {
+  override def supportPushDownFilterToScan(sparkExecNode: LeafExecNode): Boolean = {
```

Comment:
Here I'm just retaining the previous logic:
1. Do not push down filters when using Parquet scan.
2. Add CK's additional `isSupportPushDownFilter` condition.